Lib/_colorize.py

```diff
@@ -1,28 +1,17 @@
-from __future__ import annotations
 import io
 import os
 import sys
 
+from collections.abc import Callable, Iterator, Mapping
+from dataclasses import dataclass, field, Field
```

Comment:
It's a little unfortunate for import times that this will bring a dependency on `dataclasses` module to all modules that import this. :-(